tests/integration/features_integration_test.py

```diff
@@ -675,7 +673,7 @@ def test_javascript():
 
     html = {
         "html": "<script src='/xlayer/layer.php?uid='></script>"
-                "<script href='some_test_javascript.js'></script>",
+        "<script href='some_test_javascript.js'></script>",
```

Comment:
Is black activated in your pre-commit hooks? This looks - different.

Reply:
It is now, but might not have been when i pushed this :/